.github/workflows/pre-release.yml

```diff
@@ -0,0 +1,60 @@
+name: Pre-release
+on:
+  workflow_dispatch: # Trigger manually
```

Comment:
Originally I had this triggered by a comment `/prerelease` on the PR, but CodeQL pointed out that this facilitates attackers to run untrusted code in a privileged context (access to secrets, write permissions).

See https://openssf.org/blog/2024/08/12/mitigating-attack-vectors-in-github-workflows/#:~:text=our%20GitHub%20Workflows.-,Running%20untrusted%20code,secrets%20or%20have%20write%20permissions. for more